lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/optional/TestDataFormatterGenericOptional.py

```diff
@@ -79,6 +79,18 @@ def cleanup():
             ],
         )
 
+        self.expect_var_path("*number_engaged", value="42")
+        self.expect_var_path("*x", children=[ValueCheck(name="x", value="42")])
+        self.expect_var_path("x->x", value="42")
+
+        # The error message could use some improvement, but at least we can
+        # check we don't crash.
```

Comment:
Yea would be nice to bubble a nicer error up somehow. Looks like ValueObject currently just drops any error set by the dereferenced ValueObject (we don't set one in the formatter currently anyway). But not urgent